photon-ml/src/main/scala/com/linkedin/photon/ml/optimization/game/GLMOptimizationConfiguration.scala

```diff
@@ -38,16 +37,23 @@ protected[ml] case class GLMOptimizationConfiguration private (
     regularizationType: RegularizationType = RegularizationType.L2) {
 
   override def toString: String = {
-    s"maxNumberIterations: $maxNumberIterations\t" +
-      s"convergenceTolerance: $convergenceTolerance\t" +
-      s"regularizationWeight: $regularizationWeight\t" +
-      s"downSamplingRate: $downSamplingRate\t" +
-      s"optimizerType: $optimizerType\t" +
+    s"maxNumberIterations: $maxNumberIterations," +
+      s"convergenceTolerance: $convergenceTolerance," +
+      s"regularizationWeight: $regularizationWeight," +
+      s"downSamplingRate: $downSamplingRate," +
+      s"optimizerType: $optimizerType," +
```

Comment:
Oh, it's because tab separated toString method is inconsistent with some other comma separated toString methods, maybe we should leave a space behind the comma though.